scripts/apps/search/directives/RawSearch.js

```diff
@@ -1,12 +1,22 @@
-RawSearch.$inject = ['asset', '$location'];
+/**
+ * @memberof superdesk.apps.search
+ * @ngdoc directive
+ * @name RawSearch
```

Comment:
I know this might be a bit counter-intuitive, but we need to use the name that this service is [registered as](https://github.com/marwoodandrew/superdesk-client-core/blob/5eaedb32f72f050d4d8cf9a888682625b1193724/scripts/apps/search/index.js#L46) for documentation to be efficient, in this case `sdRawSearch`. It makes more sense from a documentation perspective.